app/index.js

```diff
@@ -84,146 +42,39 @@ module.exports = class extends Generator {
   }
 
   writing() {
-    this._writingModernizr();
-    this._writingGulpfile();
-    this._writingPackageJSON();
-    this._writingBabel();
-    this._writingGit();
-    this._writingEditorConfig();
-    this._writingH5bp();
-    this._writingStyles();
-    this._writingScripts();
-    this._writingHtml();
-    this._writingMisc();
-  }
-
-  _writingModernizr() {
-    if(this.includeModernizr) {
-      this.fs.copy(
-        this.templatePath('modernizr.json'),
-        this.destinationPath('modernizr.json')
-      );
-    }
-  }
-  _writingGulpfile() {
-    this.fs.copyTpl(
-      this.templatePath('gulpfile.js'),
-      this.destinationPath('gulpfile.js'),
-      {
-        date: (new Date).toISOString().split('T')[0],
-        name: this.pkg.name,
-        version: this.pkg.version,
-        includeSass: this.includeSass,
-        includeBootstrap: this.includeBootstrap,
-        includeBabel: this.options['babel'],
-        testFramework: this.options['test-framework']
-      }
-    );
-  }
-
-  _writingPackageJSON() {
-    this.fs.copyTpl(
-      this.templatePath('_package.json'),
-      this.destinationPath('package.json'),
-      {
-        includeSass: this.includeSass,
-        includeBabel: this.options['babel'],
-        includeJQuery: this.includeJQuery,
-        includeBootstrap: this.includeBootstrap,
-        includeModernizr: this.includeModernizr
-      }
-    );
-  }
-
-  _writingBabel() {
-    this.fs.copy(
-      this.templatePath('babelrc'),
-      this.destinationPath('.babelrc')
-    );
-  }
-
-  _writingGit() {
-    this.fs.copy(
-      this.templatePath('gitignore'),
-      this.destinationPath('.gitignore'));
-
-    this.fs.copy(
-      this.templatePath('gitattributes'),
-      this.destinationPath('.gitattributes'));
-  }
-
-  _writingEditorConfig() {
-    this.fs.copy(
-      this.templatePath('editorconfig'),
-      this.destinationPath('.editorconfig')
-    );
-  }
-
-  _writingH5bp() {
-    this.fs.copy(
-      this.templatePath('favicon.ico'),
-      this.destinationPath('app/favicon.ico')
-    );
-
-    this.fs.copy(
-      this.templatePath('apple-touch-icon.png'),
-      this.destinationPath('app/apple-touch-icon.png')
-    );
+    const _self = this;
+    const data = {
```

Comment:
A more meaningful variable name, for example:
```suggestion
    const templateData = {
```

Reply:
Sounds good!